homework-classes/RepoView.js

```diff
@@ -19,8 +19,38 @@
      * @param {Object} repo A repository object.
      */
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
-      console.log('RepoView', repo);
+      this.container.innerHTML = '';
+      const repoDetailsTable = createAndAppend('table', this.container, {
+        class: 'repository-item-table',
+      });
+      const firstRow = this.appendRepoDetail(
+        'Repository:',
+        '',
+        repoDetailsTable,
+      );
+      createAndAppend('a', firstRow.lastChild, {
+        href: repo.html_url,
+        text: repo.name,
+        target: '_blank',
+      });
+      this.appendRepoDetail('Description:', repo.description, repoDetailsTable);
+      this.appendRepoDetail('Forks:', repo.forks, repoDetailsTable);
+      const timeRow = this.appendRepoDetail('Updated:', '', repoDetailsTable);
+      createAndAppend('time', timeRow.lastChild, {
+        text: this.getDateTimeText(repo.updated_at),
+      });
+    }
+
+    appendRepoDetail(header, description, parentTable) {
```

Comment:
Strictly speaking, this method could be `static` as it does not reference inside its method body. The same goes for the `getDateTimeText` method. Static methods are attached to the class itself (actually, the underlying constructor function of the class). Regular methods are attached to the prototype object.

This is just for info. No need to change anything here.